configs/test/habitat_cl_example.yaml

```diff
@@ -0,0 +1,71 @@
+ENVIRONMENT:
+  MAX_EPISODE_STEPS: 500
+SIMULATOR:
+  AGENT_0:
+    SENSORS: ['RGB_SENSOR', 'DEPTH_SENSOR']
+  HABITAT_SIM_V0:
+    GPU_DEVICE_ID: 0
+  RGB_SENSOR:
+    WIDTH: 256
+    HEIGHT: 256
+  DEPTH_SENSOR:
+    WIDTH: 256
+    HEIGHT: 256
+# TASK:
+#   TYPE: Nav-v0
+#   SUCCESS_DISTANCE: 0.2
+
+#   SENSORS: ['POINTGOAL_WITH_GPS_COMPASS_SENSOR']
+#   POINTGOAL_WITH_GPS_COMPASS_SENSOR:
+#     GOAL_FORMAT: "POLAR"
+#     DIMENSIONALITY: 2
+#   GOAL_SENSOR_UUID: pointgoal_with_gps_compass
+
+#   MEASUREMENTS: ['DISTANCE_TO_GOAL', 'SUCCESS', 'SPL']
+#   SUCCESS:
+#     SUCCESS_DISTANCE: 0.2
+
+TASKS:
+  - TYPE: Nav-v0
+    SUCCESS_DISTANCE: 0.169
+    TASK_LABEL: MyTask
+    SENSORS: ['POINTGOAL_WITH_GPS_COMPASS_SENSOR']
+    POINTGOAL_WITH_GPS_COMPASS_SENSOR:
+      GOAL_FORMAT: "POLAR"
+      DIMENSIONALITY: 2
+    GOAL_SENSOR_UUID: pointgoal_with_gps_compass
+
+    MEASUREMENTS: ['DISTANCE_TO_GOAL', 'SUCCESS', 'SPL']
+    SUCCESS:
+      SUCCESS_DISTANCE: 0.2
+    DATASET:  # you can specify your own dataset per task
+      TYPE: PointNav-v1
+      SPLIT: test
+      DATA_PATH: data/datasets/pointnav/habitat-test-scenes/v1/{split}/{split}.json.gz
+  - TYPE: Nav-v0
+    SUCCESS_DISTANCE: 0.2
+
+    SENSORS: ['POINTGOAL_WITH_GPS_COMPASS_SENSOR']
+    POINTGOAL_WITH_GPS_COMPASS_SENSOR:
+      GOAL_FORMAT: "POLAR"
+      DIMENSIONALITY: 2
+    GOAL_SENSOR_UUID: pointgoal_with_gps_compass
+
+    MEASUREMENTS: ['DISTANCE_TO_GOAL', 'SUCCESS', 'SPL']
+    SUCCESS:
+      SUCCESS_DISTANCE: 0.2
+    # "global' dataset is used if unspecified
+  - TYPE: Nav-v0
+    SUCCESS_DISTANCE: 1.53
+CHANGE_TASK_BEHAVIOUR:
+  TYPE: FIXED # or RANDOM
+  CHANGE_TASK_PROB: 0.1 # for RANDOM, how likely it is to change task at defined horizon
+  # AFTER_N_EPISODES: 1   # After how many episodes we want to change task
+  AFTER_N_CUM_STEPS: 10 # After how many cumulative steps we want to change task
+  LOOP: ORDER # RANDOM for sampling a different task with uniform probability
+
+DATASET:
+  TYPE: PointNav-v1
+  SPLIT: val
+  # DATA_PATH: data/datasets/pointnav/gibson/v1/{split}/{split}.json.gz
```

Comment:
Any reason to leave this comment here?

Reply:
just wanted to show some available options in config file, I can get rid of that